NoteFind/Alerts/ErrorAlerts.h

```diff
@@ -0,0 +1,33 @@
+//
+//  ErrorAlerts.h
+//  NoteFind
+//
+//  Created by Khloe Wright on 7/22/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "LoginViewController.h"
+#import "FeedViewController.h"
+#import "RegisterViewController.h"
+#import "NewNoteViewController.h"
+#import <UIKit/UIKit.h>
+
+@class LoginViewController;
+@class FeedViewController;
+@class RegisterViewController;
+@class NewNoteViewController;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ErrorAlerts : NSObject
++ (void)loginFailure:(LoginViewController *)loginVC;
++ (void)alertVoidRequiredFields:(LoginViewController *)loginVC;
++ (void)logoutFailure:(FeedViewController *)feedVC;
++ (void)passwordNotConfirmed:(RegisterViewController *)regVC;
++ (void)retrieveNotesFailure:(FeedViewController *)feedVC;
++ (void)retrieveTagsFailure:(UIViewController *)vc;
++ (void)retrieveBooksFailure:(UIViewController *)vc;
++ (void)errorPostingNote:(NewNoteViewController *)newNoteVC;
```

Comment:
Given how these methods are actually implemented, they don't need the actual concrete types (`FeedViewController`, etc) unless the implementation depends on some API or data to be accessed from that specific object type. Given the view controller is just used for presenting the alert, I think you can get away with just passing `UIViewController` instead of a specific concrete subtype